src/lang/export.pl

```diff
@@ -44,3 +44,23 @@
   tripledb_export(Directory),
   obda_export(Directory),
   forall(memorize_hook(Directory), true).
+
+     /*******************************
+     *          UNIT TESTS          *
+     *******************************/
+
+:- begin_tests('lang_export').
+
+get_path(Path):-
+  working_directory(X,X), string_concat(X, "test_lang_export", Path).
+
+test('stores knowledge in test_lang_export directory and restores the same', 
```

Comment:
the test should check whether triples were really loaded after memorize is called